scene/3d/vehicle_body.h

```diff
@@ -40,6 +40,43 @@ class VehicleWheel : public Spatial {
 
 	friend class VehicleBody;
 
+	struct WheelXform {
+		Vector3 up;
+		Vector3 right;
+		Vector3 origin;
+		real_t steering = 0;
+	};
+
+	class FTIData {
+		real_t curr_rotation = 0;
+		real_t curr_rotation_delta = 0;
+
+	public:
+		WheelXform curr;
+		WheelXform prev;
+
+		// If a wheel is added on a frame, the xform will not be set until it has been physics updated at least once.
+		bool unset = true;
+		bool reset_queued = false;
+
+		void rotate(real_t p_delta) {
+			curr_rotation += p_delta;
+			curr_rotation_delta = p_delta;
+
+			// Wrap rotation to prevent float error.
+			double wrapped = Math::fmod(curr_rotation + Math_PI, Math_TAU);
+			if (wrapped < 0)
+				wrapped += Math_TAU;
```

Comment:
There is `Math::fposmod`.

Reply:
Wasn't aware of the function, but on looking, that version adds `0.0f` for the sign of zero, which is unnecessary here.

We could add a version of the function that doesn't do that (it seems for gdscript use), but probably for another PR.